corrector.rb

```diff
@@ -0,0 +1,6 @@
+class Corrector
+  def correct_name(name)
+    name = name.capitalize
+    name[0, 10] if name.length > 10
```

Comment:
- [ ] Correct_name is not working as expected. If you pass a name with `name.length > 10` the function returns the name trimmed and capitalized if you pass a name lesser than 10 characters it will return `nil`. Kindly fix the issue.

Reply:
Issue resolved